railties/lib/rails/generators/rails/credentials/credentials_generator.rb

```diff
@@ -18,24 +18,25 @@ def add_credentials_file
           say template, :on_green
           say ""
 
-          add_credentials_file_silently(template)
+          add_credentials_file_silently(template: template)
 
           say "You can edit encrypted credentials with `bin/rails credentials:edit`."
           say ""
         end
       end
 
-      def add_credentials_file_silently(template = nil)
+      def add_credentials_file_silently(template: credentials_template, config_path: nil, key_path: nil)
+        credentials = credentials(config_path, key_path)
         unless credentials.content_path.exist?
-          credentials.write(credentials_template)
+          credentials.write(template)
         end
       end
 
       private
-        def credentials
+        def credentials(config_path = nil, key_path = nil)
           ActiveSupport::EncryptedConfiguration.new(
-            config_path: "config/credentials.yml.enc",
-            key_path: "config/master.key",
+            config_path: config_path || "config/credentials.yml.enc",
+            key_path: key_path || "config/master.key",
```

Comment:
Is my understanding correct that your intention of this feature is to make the path to the credentials file configurable?

Reply:
It is already configurable: https://edgeapi.rubyonrails.org/classes/Rails/Application.html#method-i-credentials

However the `bin/rails credentials:edit` helper doesn't respect it while the rest of Rails does. It will be cool to align the helper behavior to actual Rails like OP wants. 👍

Reply:
Yep, that's the basic idea. hartator summed it up well. 

Despite the lack of recent activity, I am still interested in this. I've just merged main to get it back up to date and show some signs of life so this doesn't closed due to staleness. I'm also working on an alternate approach to try to address some of the issues mentioned in the review above. Should have something ready soon...

Reply:
WE WANT IT!